src/account.ts

```diff
@@ -102,6 +103,12 @@ export default class Account {
         this.client = client;
 
         const bulkConfigUrl = `./assets/zkbob-${NETWORK}-coldstorage.cfg`
+
+        let delegatedProverUrl: string | undefined = undefined;
+        if (DELEGATED_PROVER_URL) {
+            delegatedProverUrl = DELEGATED_PROVER_URL;
```

Comment:
Do you process the case when `DELEGATED_PROVER_URL` is a void string? It seems there are no way to pass in the library undefined value except you remove the associated string from the `.env` file

Reply:
I agree it is not necessary since `zkbob-client-js` handles void strings correctly. Removed it in https://github.com/zkBob/zkbob-console/pull/38/commits/f25f61a96d9bb09cc5f2450a338dd2390140850b.